docs/support-contribution.md

```diff
@@ -0,0 +1,13 @@
+# Support and Contribution
+
+## Kyma Support
+
+Here are our support channels:
+
+- [Kyma Slack channels](http://slack.kyma-project.io/)
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/kyma)
+- If you need to report a bug or have a feature request, raise an issue in the given repository using one of the available templates.
```

Comment:
```suggestion
- If you need to report a bug or have a feature request, raise an issue in the given module repository using one of the available templates.
```
Maybe we could also say that for general Kyma-related issues, the user should create an issue in the `/kyma` repo and add that link -> https://github.com/kyma-project/kyma/issues/new/choose (having a direct link is quite beneficial for the users I guess).